### ISSUE TITLE
Evaluate PICMI standard

### ISSUE THREAD

**Issue by cenit** [ALaDyn/ALaDyn#73]
https://picmi-standard.github.io/
We should keep this idea monitored and decide if it is interesting to implement it also in ALaDyn

**Comment by titoiride** [comment]
I agree. As soon as more informations will be available, we can decide if to implement the PICMI conventions in ALaDyn. In the meantime, as a step towards it, we should think about a python-based input management.

**Comment by cenit** [comment]
PICMI standard never evolved and ALaDyn is under minimum maintenance. Closing issue
